project_task_reviewer/models/project_task.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2018 RGB Consulting - <odoo@rgbconsulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    reviewer_id = fields.Many2one(comodel_name='res.users', string='Reviewer',
+                                  default=lambda self: self.env.uid,
+                                  index=True, track_visibility='always')
+
+    @api.model
+    def _message_get_auto_subscribe_fields(self, updated_fields,
```

Comment:
Is this needed? Because you already set track_visibility on the field to 'always'.

Reply:
Hi,
Yes, it's needed because **track_visibility='always'** only adds the field value in the chatter. 
On the other hand the method auto subscribes the reviewer to the task.